packages/server/src/fhir/routes.ts

```diff
@@ -155,9 +158,8 @@ protectedRoutes.post(
 protectedRoutes.use(
   '*',
   asyncWrap(async (req: Request, res: Response) => {
-    const router = new FhirRouter();
+    router = new FhirRouter({ introspectionEnabled: getConfig().introspectionEnabled });
```

Comment:
We should either:
1. Go back to `const`, or
2. Do proper lazy load

The lazy initialization is intended to only create a `new FhirRouter` once.  I'm ok to move that to a separate commit though.

Lazy initialization would look like this:

```ts
if (!router) {
  router = new FhirRouter({ introspectionEnabled: getConfig().introspectionEnabled });
}
```